torch/csrc/cuda/utils.cpp

```diff
@@ -23,14 +23,14 @@ THPUtils_PySequence_to_CUDAStreamList(PyObject* obj) {
   for (Py_ssize_t i = 0; i < length; i++) {
     PyObject* stream = PySequence_Fast_GET_ITEM(seq.get(), i);
 
-    if (PyObject_IsInstance(stream, THCPStreamClass)) {
```

Comment:
This change allows both `torch.Stream` and `torch.cuda.Stream` to be reinterpreted as `CUDAStream` in cpp.